modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -9,7 +9,7 @@ You can use the Bare Metal Operator (BMO) to provision bare-metal hosts in a use
 
 [NOTE]
 ====
-To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
+Provisioning new bare-metal hosts to the cluster using the BMO sets the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false` by default.
```

Comment:
I think my [previous comment](https://github.com/openshift/openshift-docs/pull/77264#discussion_r1798618089) got lost. It's critical to point out that the user **must not** set `spec.externallyProvisioned` to `true`, which is a common thing that happens.